app/assets/stylesheets/course/forum/posts.scss

```diff
@@ -0,0 +1,5 @@
+.course-forum-posts {
+  &.reply {
+    @include forum-post;
```

Comment:
If you have JavaScript, you will not be going to the reply page. The current forum's reply composition form will appear directly under the post. The reply page is for less-modern browsers.